File: Hilos/OtraClase.py
class OtraClase:
    def __init__(self):
        self._mensaje = "Mensaje desde la clase base"

    def get_mensaje(self):
        return self._mensaje

    def set_mensaje(self, mensaje: str):
        self._mensaje = mensaje

    def imprimir_mensaje(self):
        print(self._mensaje)

File: Hilos Java/Hilos/otraClase.java
public class otraClase {
    private String mensaje;

    public otraClase(){
        this.mensaje = "Mensaje desde la clase base";
    }

    public String getMensaje() {
        return mensaje;
    }

    public void setMensaje(String mensaje) {
        this.mensaje = mensaje;
    }
     public void imprimirMensaje(){
        System.out.println(mensaje);
     }
}
